Formula/wxpython.rb

```diff
@@ -18,12 +18,12 @@ class Wxpython < Formula
   end
 
   depends_on "freetype"
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
   depends_on "libpng"
   depends_on "libtiff"
   depends_on "numpy"
   depends_on "pillow"
-  depends_on "python@3.9"
+  depends_on "python@3.10"
```

Comment:
May need to revert this. Arch bug mentions an issue in current stable that is incompatible with `python@3.10`: https://bugs.archlinux.org/task/73763?project=5&string=python-wxpython